timApp/static/scripts/tim/plugin/calendar/calendar.component.ts

```diff
@@ -1137,6 +1137,7 @@ export class TimCalendarModule implements DoBootstrap {
     ngDoBootstrap(appRef: ApplicationRef): void {}
 }
 
+// pluginMap.set("tim-calendar", CalendarComponent);
```

Comment:
Sama näyttää olevan importdatan kanssa. Oliko tämän ja sen kanssa jotain ongelmia?

Reply:
Testasin nyt tämän itse. Ongelmana on se, että `createComponent`:lle pitäisi vaikeissa tapauksissa antaa myös moduuliviite (`NgModuleRef`) jotta kaikki riippuvuudet toimivat. Laitoin omat kokeilut kalenterin kanssa talteen:

https://github.com/TIM-JYU/TIM/commit/f085530424f79580c32b4afcfd48578f9ee62ce7

Tähän PRään tuota ei kannata tehdä, koska siinä pitäisi vähän miettiä tuo pluginMapin sisältö uusiksi (jotta se sisältää moduulin + pluginit). Eli toistaiseksi tämä nyt olkoon ihan näin.

Mutta nyt pitäisi sitten testata muut pluginit sen varalta, että niissä ei ole vastaavaa mahdollista ongelmaa. Jos on, pitää kai toteuttaa joku moduulin lataus jo tähän.

Reply:
Oikeastaan calendarin ja muut jätin pois tuosta pluginMapista siksi että niillä ei ollut pluginLoaderia ollenkaan, ja ne eivät latautuneet. Tosin tuolla `abtype: none`lla tuon saisi näköjään kierrettyä.